source/how-tos/windows_dev_vm.rst

```diff
@@ -0,0 +1,106 @@
+.. SPDX-FileCopyrightText: © 2021 James C. Womack <J.C.Womack@bristol.ac.uk>
+   SPDX-License-Identifier: CC-BY-SA-4.0
+
+Windows development VMs
+#######################
+
+.. admonition:: What is this page about?
+
+   How to set up a Windows development virtual machine
```

Comment:
Perhaps add "... on MacOS" to the end here to clarify the audience. I see that this page could be expanded to cover doing this on Linux (or even on Windows) as well in the future, but for now it is not.

Reply:
That seems sensible. Yes, I was thinking this how-to could be expanded for other hosts.